seq_analysing_utilite/seq_analysing_utilite.py

```diff
@@ -0,0 +1,137 @@
+import os.path
+from typing import Dict, Union, List, Tuple
+
+
+
+from utilites.nucleotides_act import transcribe, complement, reverse, reverse_complement, gc_cont
+from utilites.filtering import gc_bounds_mod, length_bounds_mod, quality_threshold_mod
+
+
+def run_dna_rna_tools(*args: Union[str, str]) -> Union[str, List[str]]:
+    """
+    run_dna_rna_tools takes as input seq and operations for analysing and doing some actions with nucleotides.
+
+    Args
+    seq is a any str. You can use any count of seq and choose either upper or lower cases.
+    seq must be a DNA or RNA sequence. if you will use T and U in the same seq, run_dna_rna_tools gives you an error.
+    operation is an actions for seq.
+    operation must be "transcribe", "complement", "reverse", "reverse_complement" or "gc_cont". Otherwise run_dna_rna_tools backs with the error.
+
+    Returns
+    [str] of nucleotide after operations.
+    """
+
+    seq = args[:-1]
+    operation = args[-1]
+    dna_tools_res = []
+    for s in seq:
+        if ("T" in s or "t" in s) and ("U" in s or "u" in s):
+            return "Error"
+    if operation == "transcribe":
+        fn = transcribe
+    elif operation == "complement":
+        fn = complement
+    if operation == "reverse":
+        fn = reverse
+    elif operation == "reverse_complement":
+        fn = reverse_complement
+    elif operation == "gc_cont":
+        fn = gc_cont
+    for s in seq:
+        result = fn(s)
+        dna_tools_res.append(result)
+    return dna_tools_res if len(dna_tools_res) > 1 else dna_tools_res[0]
+
+
+def filter_fastq(input_fastq:str,
+                 output_fasta: str,
+                 gc_bounds: Tuple[float,float] = (0.0, 100.0),
+                 length_bounds:Tuple[int,int] = (0, 2**32),
+                 quality_threshold: float = 0.0) -> None:
+    """
```

Comment:
I generally like the docstrings in this repo, they are very explicit. I will use this approach in future, too.